.github/workflows/tests.yml

```diff
@@ -0,0 +1,89 @@
+name: CI tests
+
+defaults:
+  run:
+    shell: bash
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-20.04]
+
+    container: 
+        image: koreader/koappimage:0.2.2
+        options: --user ko
```

Comment:
I think this is part of what I couldn't find. :100:

Reply:
yup, I just discovered it in one of the numerous issues in github/runner